CHANGELOG.md

```diff
@@ -1,9 +1,64 @@
 Changelog
 =========
 
+0.4.3
+-----
+
+**Date:** 2019-02-14
+
+System Updates and Minor Bug Fixes
+
+This release adds updates and new HPC system templates. Important bug
+fixes include I/O work-arounds for issues in OpenMPI 2.0-4.0 (mainly
+with HDF5), guards for particle creation with user-defined
+profiles, a fixed binomial current deposition, checks for the number
```

Comment:
I guess its a matter of taste, but I would write `fixed binomial current smoothing`